test/common/stats/thread_local_store_test.cc

```diff
@@ -31,10 +32,9 @@ class TestAllocator : public RawStatDataAllocator {
   ~TestAllocator() { EXPECT_TRUE(stats_.empty()); }
 
   RawStatData* alloc(const std::string& name) override {
-    std::unique_ptr<RawStatData>& stat_ref = stats_[name];
+    CSmartPtr<RawStatData, freeAdapter>& stat_ref = stats_[name];
```

Comment:
you can't just pass `:free` directly here?

Reply:
It wouldn't compile because ::free takes a void*, CSmartPtr wants a function that takes RawStatData*